log/logger.py

```diff
@@ -188,7 +188,7 @@ def step_info(self, step_name):
 
     def get_additional_logs(self):
         from core.test_run import TestRun
-        from test_tools.fs_utils import check_if_file_exists
+        from test_tools.fs_tools import check_if_file_exists
```

Comment:
It needs to be imported here because fs_tools imports TestRun which imports logger and there we have circular dependencies. I think that core functionalities refactor to avoid circular dependencies will be done in next step in other PR